x-pack/plugins/synthetics/public/plugin.ts

```diff
@@ -271,7 +271,7 @@ function registerSyntheticsRoutesWithNavigation(
                     defaultMessage: 'Monitors',
                   }),
                   app: 'synthetics',
-                  path: MONITORS_ROUTE,
+                  path: OVERVIEW_ROUTE,
```

Comment:
Also updated the route for the observability sidebar to point to Synthetics Overview by default, which is reflected in the figma.